repo/db/utxos_test.go

```diff
@@ -32,11 +26,39 @@ func init() {
 	}
 }
```

Comment:
This wants to be a factory that lives in `test/factory/utxo.go`. Example: https://github.com/OpenBazaar/openbazaar-go/blob/master/test/factory/sale_record.go

Reply:
Completed in https://github.com/OpenBazaar/openbazaar-go/pull/1265/commits/d2a28fdc497decc195647cc38031e38b381116d1